guides/7-jlp/2-How-JLP-Works.md

```diff
@@ -77,9 +54,13 @@ It is essential to note that pool earnings and losses (index token appreciation/
 | ------------------ | -------------------------------------------------------------------------- |
 | Opening a Position | 6 BPS                                                                     |
 | Closing a Position | 6 BPS                                                                     |
-| Price Impact Fee   | Variable (see [reference](../perpetual-exchange/trading#price-impact-fee)) |
+| Price Impact Fee   | Variable (see [reference](../8-perpetual-exchange/2-how-it-works.md#price-impact-fee)) |
 | Swap Fee           | Between 0 BPS to 150 BPS depending on pool weightage                       |
-| Borrow Rate        | 1 BPS/hour x token utilization percentage                                  |
+| Borrow Rate        | 0.8 BPS/hour x token utilization percentage                                  |
```

Comment:
Sorry I should have mentioned this earlier; can we update this to add 1 BPS / hour for stablecoins, in addition to the `0.8 BPS/hour` we have now?

Reply:
I previously added this as an info portion below the table so the table wouldn't have too much words on it - is it ok?